README.md

```diff
@@ -32,6 +32,7 @@ symfony_composer_path: "{{ ansistrano_deploy_to }}/composer.phar"
 symfony_composer_options: '--no-dev --optimize-autoloader --no-interaction'
 symfony_composer_self_update: true # Always attempt a composer self-update
 symfony_composer_version: 1.10.1 # Install specific composer version. If this variable is not set the latest stable version is installed
+symfony_composer_channel: 2.2 # Install specific composer channel. 
```

Comment:
It would be helpful to append `(see https://getcomposer.org/download/#download-channels)` because this is a somewhat advanced topic.